doc/lua_api.txt

```diff
@@ -453,6 +459,10 @@ the global `minetest.registered_*` tables.
 * `minetest.register_craftitem(name, item definition)`
     * added to `minetest.registered_items[name]`
 
+* `minetest.unregister_item(name)`
+    * erased from minetest.registered_items[] and from item associated table
```

Comment:
Tell me*, you mean.  Not "say me".
When "say" is used with an indirect object such as "me" (i.e. in dative form), you must use the preposition "to" before the object.  "Please say to me" would be grammatical at least, but awkward.  When using "say" imperatively and your sentence has an indirect object, use "tell" instead.

Anyway, to respond to your actual comment:
Why not something like "Unregisters the item `name` from engine, and deletes the entry with key `name` from `minetest.registered_items`."